en/computer_vision/collision_prevention.md

```diff
@@ -213,7 +214,104 @@ The diagram below shows a simulation of collision prevention as viewed in Gazebo
+
+   ```sh
+   - topic: /fmu/out/obstacle_distance_fused
+     type: px4_msgs::msg::ObstacleDistance
+   ```
+
+   For more information see [DDS Topics YAML](../middleware/uxrce_dds.md#dds-topics-yaml) in _uXRCE-DDS (PX4-ROS 2/DDS Bridge)_.
+
+3. Open PlotJuggler and navigate to the **Tools > Reactive Script Editor** section.
+   In the **Script Editor** tab, add following scripts in the appropriate sections:
+
+   - **Global code, executed once:**
+
+     ```lua
+     obs_dist_fused_xy = ScatterXY.new("obstacle_distance_fused_xy")
+     obs_dist_min = Timeseries.new("obstacle_distance_minimum")
+     ```
+
+   - **function(tracker_time)**
+
+     ```lua
+     obs_dist_fused_xy:clear()
+
+     i = 0
+     angle_offset = TimeseriesView.find("/fmu/out/obstacle_distance_fused/angle_offset")
+     increment = TimeseriesView.find("/fmu/out/obstacle_distance_fused/increment")
+     min_dist = 65535
+
+     -- Cache increment and angle_offset values at tracker_time to avoid repeated calls
+     local angle_offset_value = angle_offset:atTime(tracker_time)
+     local increment_value = increment:atTime(tracker_time)
+
+     if increment_value == nil or increment_value <= 0 then
+         print("Invalid increment value: " .. tostring(increment_value))
+         return
+     end
+
+     local max_steps = math.floor(360 / increment_value)
+
+     while i < max_steps do
+         local str = string.format("/fmu/out/obstacle_distance_fused/distances[%d]", i)
+         local distance = TimeseriesView.find(str)
+         if distance == nil then
+             print("No distance data for: " .. str)
+             break
+         end
+
+         local dist = distance:atTime(tracker_time)
+         if dist ~= nil and dist < 65535 then
+             -- Calculate angle and Cartesian coordinates
+             local angle = angle_offset_value + i * increment_value
+             local y = dist * math.cos(math.rad(angle))
+             local x = dist * math.sin(math.rad(angle))
+
+             obs_dist_fused_xy:push_back(x, y)
+
+             -- Update minimum distance
+             if dist < min_dist then
+                 min_dist = dist
+             end
+         end
+
+         i = i + 1
+     end
+
+     -- Push minimum distance once after the loop
+     if min_dist < 65535 then
+         obs_dist_min:push_back(tracker_time, min_dist)
+     else
+         print("No valid minimum distance found")
+     end
+     ```
+
+4. Enter a name for the script on the top right, and press **Save**.
+   Once saved, the script should appear in the _Active Scripts_ section.
+5. Start streaming the data using the approach described in [Plotting uORB Topic Data in Real Time using PlotJuggler](../debug/plotting_realtime_uorb_data.md).
+   You should see the `obstacle_distance_fused_xy` and `obstacle_distance_minimum` timeseries on the left.
+
+Note that to run the script again after clearing the data, you have to press **Save** again.
```

Comment:
@Claudio-Chies This looks excellent. I've subedited to improve the structure a little, and made the video link to youtube.

This one line confuses me. Why have you cleared the data? (first time this is mentioned). Where are we pressing Save? Are you saying that we need to open our Lua scripts and save them again?